### PR TITLE
Fixes #1620. Make handbuilt lexer to avoid cyclic dependence of tool and plugin.

### LINKED ISSUE #1620: XPathLexer not updated in release process
Similar to #796 the XPathLexer.class was built with 4.5.3 instead of 4.6.  As a result the following error message will result whenever the XPath libraries are used:

"ANTLR Tool version 4.5.3 used for code generation does not match the current runtime version 4.6"

### PR DESCRIPTION
**Pull request by parrt**

### CODE REVIEW

File: runtime/Java/src/org/antlr/v4/runtime/tree/xpath/XPathLexer.java [hunk 1]
```diff
@@ -0,0 +1,174 @@
+package org.antlr.v4.runtime.tree.xpath;
```
Comment (sharwell) [inline runtime/Java/src/org/antlr/v4/runtime/tree/xpath/XPathLexer.java]: :bulb: copyright header

**Comment by sharwell** [comment]
I prefer the approach of not having `RuntimeMetaData` to break every release, but given that limitation this approach seems reasonable.

**Comment by KvanTTT** [comment]
Where is `channelNames`?

**Comment by parrt** [comment]
hahah. not needed for this little lexer.

**Pull request merged**